### ISSUE TITLE
HELP!!!!

### ISSUE THREAD

**Issue by shinianzhihou** [abojchevski/graph2gauss#8]
I am training the node classification on my dataset.

The shape of A is [14000,14000]. X is [14000,240].

I set L=64, p_val=0.0, p_test=0.0.

BUT the loss keep nan during the training process.

Have tried changed the learning rate from 1 to 1e-8.

Plz give some suggestions or ideas.

orz

**Comment by shinianzhihou** [comment]
Solved.
Not normalize to [0,1].

~~~~
